R/R.sublime-syntax

```diff
@@ -83,10 +98,11 @@ contexts:
       scope: punctuation.terminator.semicolon.r
 
   storage-types:
-    - match: \b(character|complex|double|expression|integer|list|logical|numeric|single|raw)\b(\s*\()
+    - match: \b(character|complex|double|expression|integer|list|logical|numeric|single|raw)\s*(?=\()
+      scope: meta.function-call.r
       captures:
         1: storage.type.r
```

Comment:
Don't we need `meta.function-call.name` here as well? At least that's how I understood your RFC.

Same for all similar occurances later in the file (`builtin-fuctions` context).

Reply:
We can of course add it, of course. May even be better for consistency. Just didn't add it here as I did not expect sophisticated expressions to be valid for function name construction in R - but I am not a R expert though.

Reply:
I think it just makes sense to apply the same scopes for all syntaxes, even if they don't have a particular construct that would require it. I mean, unification of scope names is our ultimate goal.